lib/pretty/style.ml

```diff
@@ -1,15 +1,44 @@
-type t = ANSITerminal.style list
-
-let fg_black = ANSITerminal.[ Foreground Black ]
-let fg_white = ANSITerminal.[ Foreground White ]
-let repo = ANSITerminal.[ Bold; blue ]
-let selected = ANSITerminal.[ Bold; green ]
-let directory = ANSITerminal.[ Bold; magenta ]
-let chosen = ANSITerminal.[ Bold; magenta ]
-let secondary = ANSITerminal.[ cyan ]
-let bold = ANSITerminal.[ Bold ]
-let issue_open = ANSITerminal.[ green ]
-let issue_closed = ANSITerminal.[ red ]
-let pr_closed = ANSITerminal.[ Bold; red ]
-let pr_open = ANSITerminal.[ Bold; green ]
-let pr_merged = ANSITerminal.[ Bold; magenta ]
+module Color = Ansifmt.Color
+module Styling = Ansifmt.Styling
+
+type t = Styling.t
+
+let ( & ) = Styling.( & )
+
+(* Empty style *)
+let none = Styling.none
+
+(* Repository name. *)
+let repo = Styling.(bold & fg Color.blue)
```

Comment:
It would be nice to write just `Styling.(bold & fg blue)` without the `Color` but I don't really want to open `Color`.

Eh, fine for now, not a big deal.

Reply:
Makes sense, perhaps we should re-export some of the common colors in `Styling`

Reply:
Actually, I take it back. I'm doing the same `Color.name` in my website written in Elm:

* https://github.com/chshersh/chshersh.github.io/blob/417529c96192109df6205f45d455687cf1ce5ac2/src/Page/Main.elm#L55

I guess, words like `yellow` or `magenta` have the only implicit meaning of color. But for more exotic colors like "suva grey" or "gainsboro" it's not that clear that it's a color. So having a `Color.` prefix is better.

I'll keep the prefix everywhere for now for consistency. Usually consistency is superior to typing fewer characters occasionally.

So I wouldn't push for reexporting colors. In GitHub TUI, I'll probably introduce some specific colors locally for different themes, so it's not a problem at all.

Reply:
Fair enough, I was a bit skeptical anyway